grails-app/domain/org/pih/warehouse/order/Order.groovy

```diff
@@ -109,6 +112,21 @@ class Order implements Serializable {
         return status
     }
 
+    def getDisplayStatus() {
+        def shipments = listShipments()
```

Comment:
I just realized that this is already there, I'll rewrite my pr (https://github.com/openboxes/openboxes/pull/1518)

Reply:
@awalkowiak i saw this comment but thought you had made it in one of your PRs. i didn't realize you noticed it in my PR :laughing:

Reply:
I noticed it here, but I refactored this in my. there will be need for veeery small rebase in this or in my PR after merge (depends on which will be merged first)